ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -338,7 +338,15 @@ if [ ! -z "${OPENSHIFT_INSTALL_PROMTAIL_ON_BOOTSTRAP:-}" ]; then
 fi
 
 date "+%F %X" > "${SHARED_DIR}/CLUSTER_INSTALL_START_TIME"
-TF_LOG=debug openshift-install --dir="${dir}" create cluster 2>&1 | grep --line-buffered -v 'password\|X-Auth-Token\|UserData:' &
+TF_LOG_PATH="${dir}/terraform.txt"
+TF_LOG_CORE="DEBUG"
+TF_LOG_PROVIDER="DEBUG"
+
+export TF_LOG_PATH
+export TF_LOG_CORE
+export TF_LOG_PROVIDER
+
+openshift-install --dir="${dir}" create cluster 2>&1 | grep --line-buffered -v 'password\|X-Auth-Token\|UserData:' &
```

Comment:
1. I think we need to compress the TF log output
2. We are scrubbing sensitive data here. Perhaps we want to scrub and compress in `populate_artifact_dir()`